.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -10,6 +10,10 @@ name: Week 1 Step 3
 Week 1 Step 3 ⬤⬤⬤◯◯◯◯◯ | 🕐 Estimated completion: 10-15 minutes
```

Comment:
This issue is a bit too long, which can overwhelm beginners. Try balancing the amount of content in drop downs and content that is outside of them, but don't put *everything* in drop downs (I've made that mistake before)